### DISCUSSION TITLE
S30 vs S40 (Apple Home plus Home Assistant)

### DISCUSSION THREAD

**Discussion by mitchkeller** [PeteRager/lennoxs30#250] [General]
Hi folks! Hoping someone can share some experiences with me as I am about to replace my HVAC system. My contractor has proposed the S40, which I was surprised to see doesn't support Apple Home (nee HomeKit) while the S30 does. I'm mainly an Apple Home guy, but I have gotten Home Assistant up and running recently and am liking how I can build more complex automations in it than I could in Apple Home. That said, I still lean toward things that are Apple Home-compatible (or Matter compatible). Contractor seemed confident he could still get the S30 and that there's not huge differences between the two. However, if folks here have been having good experiences with this integration and the S40, I might just go with that. I have a small townhouse, so I don't really need the remote sensors for averaging and such, but the contractor did indicate that the S40 has resolved some reliability problems they were having with the S30 (unrelated to smart home stuff). 

Could anyone with an S40 chime in on how it's working? I've been through the issue tracker and it appears that folks have really contributed to resolve the teething problems, but I'm curious on if folks feel they're getting a robust enough experience with the S40 and this integration or if I should get the S30 instead. I'd probably use the HomeKit integrations to be able to have the S30 in both Home Assistant and Apple Home, if I went that way.

Thanks!

**Comment by PeteRager** [comment]
That's a good question.

The reason I created this integration is because Apple HomeKit would not allow me to turn the circulation fan on and I really wanted to use the fan without running the AC or HEAT.  Also Apple Homekit would not work with more than 1 zone. According to Lennox these were problems with Apple.

One of my favorite features of the integration is the ability to dynamically change the fan CFM rate.  And the direct energy monitoring of the outdoor unit.

I have an S30, which has been reliable.  But I'd consider an S40 as they have simplified by removing the smart hub.  In my 1 zone system there are two pieces of hardware the thermostat panel and the smart hub, with the S40 there is just the thermostat panel.  So that's 1 less device to cause problems 😀.

**Reply by mitchkeller** [reply]
That's really useful information, @PeteRager! I think it may be that the smart hub was part of what my contractor was glad to have removed from the system when going from the S30 to the S40. I currently use the ecobee app to run the fan independently of heating/cooling (necessary right now as the AC part is on its last legs, at least until they bring me a temporary replacement condenser fan motor next week), but now that you mention it, the idea of being able to use this integration to control the fan is particularly appealing. I already have some automations that crank up the ceiling fans when the air purifiers detect degraded air quality, and could see adding in something through the HVAC for greater circulation being a nice add-on.

Will definitely talk with my guy closer to install time about S30 vs S40 and am leaning toward the S40 right now.

**Reply by PeteRager** [reply]
I'd be surprised if they removed HomeKit integration.  The SW in the device originated from the S30, and that's why the integration basically worked for the S40 with only a couple of minor changes.   Perhaps an S40 user can check on their thermo to see if the HomeKit setup screen is still there.

**Reply by mitchkeller** [reply]
I've looked in the PDF manuals available online really carefully, and the S30 has instructions for HomeKit pairing but the S40 manual has completely removed them, which really surprised me. Would be very interested to know if an S40 owner such as @GabeDeBacker, @julietalpha1, or @sailorgallifrey could check the menus to see if there's still a HomeKit setup screen that is just no longer in the documentation or if it's been totally scrubbed.

**Reply by 42Network** [reply]
I can confirm there are no entries in any user settings on my S40 for HomeKit, nor are there any mentions of it in the User guide or Install guide. Unfortunate, although I myself am not a HomeKit user. Is there someway to leverage the S40->HA integration to enable HomeKit with some other HA package?

**Reply by mitchkeller** [reply]
Thanks for confirming. I'm pretty sure that the HomeKit integration for HA that bridges HA devices into Apple Home would be able to expose the S40 in the Home app to do basic things, which would be good enough for me, I think. I tend to prefer writing automations in HA now but using the Home app to actually control things because the Home app comes with a nice enough user interface for me out of the box and I don't have to invest major time into building my ideal HA dashboards. (I'll probably do that eventually, but it's not a priority for me at the moment.)

**Reply by owenfi** [reply]
I'm a HomeKit user but have never paired my S30 with HomeKit directly. Rather I use this integration in HA, and HA's bridged HomeKit integration to expose it to the Home app (along with a bunch of lights). It works great–including exposing my 3 zones as separate thermostats! (I haven't used an S40, but am certain it would work similarly.)

**Reply by mitchkeller** [reply]
The integration does the trick with the S40. I had some issues with getting connected initially, but once I got the integration connected, everything has been working fine. Right now I've got schedules set in the S40 via the Lennox app and am not really using any automations in HA or Apple Home, but I do open the Home app to change settings rather than the Lennox app.
